plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/README.md

```diff
@@ -0,0 +1,119 @@
+# ProductTemplates
+
+The `Automattic\WooCommerce\Admin\Features\ProductBlockEditor\ProductTemplates` namespace contains interfaces for interacting with product editor templates, which are used to define the structure of the product editor form.
+
+General interfaces for interacting with block templates are located in the
+[`Automattic\WooCommerce\Admin\BlockTemplates`](../../../BlockTemplates/README.md) namespace.
+
+## Usage
+
+For more examples and best practices, please see the Product Editor Development Handbook.
+
+### Adding a new group to product editor templates after an existing group
+
+```php
+function YOUR_PREFIX_add_group( BlockInterface $general_group ) {
```

Comment:
Since we're using `BlockInterface`, maybe we could add the `use (...)` keyword, which would be needed to make this code work, right? If that's the case, we should update all snippets. I'm pretty sure I copied a snippet at some point with a function signature and it didn't work because the interface was not imported. I ended up deleting the explicit type and then it worked.